makeflow/src/makeflow.c

```diff
@@ -1571,7 +1571,7 @@ int main(int argc, char *argv[])
 	batch_queue_set_option(remote_queue, "priority", priority);
 	batch_queue_set_option(remote_queue, "keepalive-interval", work_queue_keepalive_interval);
 	batch_queue_set_option(remote_queue, "keepalive-timeout", work_queue_keepalive_timeout);
-	batch_queue_set_option(remote_queue, "caching", cache_mode ? "yes" : "no");
+	batch_queue_set_option(remote_queue, "disable-cache", cache_mode ? "yes" : "no");
```

Comment:
Why change to disagle-cache? I rather have enable-cache ? yes, no. It is easier to think without the double negation.